frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature

```diff
@@ -152,3 +152,64 @@ Feature: Perform Actions on repository
               And user clicks Create GitHub App button in Create GitHub App page
              Then user will be redirected to GitHub App details
               And user will see App Name as "pac-app123", App Link as "https://github.com/apps/pac-app123" and Secret as "pipelines-as-code-secret" in "openshift-pipelines" namespace
+
+
+        @regression @odc-6461 @manual
+        # Manual test case as pipeline 1.8 is not yet available
+        Scenario Outline: Add a Git repository to pipeline as code using webhook url: P-11-TC14
+            Given user has installed OpenShift Pipelines Operator version 1.8 with catalog source "<catalog_yaml>" and image content policy "<image_content_policy_yaml>"
+              And user is at repositories page
+             When user clicks on Create Repository button
+              And user enters Git Repo URL of repository as "<repository_url>"
+              And user enters Name of repository as "<repository_name>"
+              And user clicks on Show configuration options
+              And user clicks on Generate in Webhook secret as "<repository_name>"
+              And user clicks on Add button
+             Then user will be redirected to steps to configure Git repository page
+
+        Examples:
+                  | catalog_yaml                             | image_content_policy_yaml           | repository_url              | repository_name |
+                  | testData/installPipelineOperator1.8.yaml | testData/imageContentPolicy1.8.yaml | https://github.com/testing/ | git-testing     |
+
+
+        @regression @odc-6461 @manual
+        Scenario Outline: Triggering a pipeline run in added Git repository to pipeline as code using webhook url: P-11-TC15
+            Given user has installed OpenShift Pipelines Operator version 1.8 with catalog source "<catalog_yaml>" and image content policy "<image_content_policy_yaml>"
+              And user has copied webhook url and webhook secret
+              And user is on steps to configure Git repository page with its github repo
+            #   Follow scenario P-11-TC14 to be redirected to configure git repository steps
+             When user clicks on Add webhook in repository Settings
+              And user pastes the webhook url to Payload URL
+              And user pastes webhook secret to Secret
+              And user disables SSL verification
```

Comment:
This step may not be required if SSL certificates are configured, maybe add it as comment.
```suggestion
              And user disables SSL verification
```

Reply:
Thanks, added the comment